javascript/javascript1/week2/jsweek2.js

```diff
@@ -0,0 +1,103 @@
+// Flight booking fullname function
+function getFullname(firstname, surname, useFormalName) {
+    if (useFormalName) {
+        return "Lord " + firstname + " " + surname;
+    } else {
+        return firstname + " " + surname;
+    }
+}
+const fullname1 = getFullname("Benjamin", "Hughes", true);
+const fullname2 = getFullname("Benjamin", "Hughes", false);
+
+console.log(fullname1);
+console.log(fullname2);
+
+
+
+// Event application
+const wDays7 = [
+    "Thursday",
+    "Friday",
+    "Saturday",
+    "Sunday",
+    "Monday",
+    "Tuesday",
+    "Wednesday",
+];
+function getEventWeekday(daysOfEvent) {
+    const fullD = new Date();
+    const manyDaysFromToday = new Date(
+        fullD.getDate(),
+        fullD.getMonth() + daysOfEvent
+    );
+    return wDays7[manyDaysFromToday.getDay()];
+}
+// With todays weekday a tuesday
+console.log(getEventWeekday(9)); // Logs out "Thursday"
+
+// With todays weekday a Friday
+console.log(getEventWeekday(5)); // Logs out "Sunday"
+
```

Comment:
OBS! 😊

You have a really good approach, and you are almost there, but there are some things to reconsider in this implementation. 💡

I like to inspect the values I am working with, to fully understand my implementation. Try to console.log() the following lines before your return statement in the getEventWeekday function;
    console.log(fullD.getDate());
    console.log(fullD.getMonth());
    console.log(fullD.getMonth() + daysOfEvent);
    console.log(manyDaysFromToday.getDay());

Investigate these values, and reflect on how you apply the values in your current implementation 🧐
There is a hint in the assignment description to the remainder operator. How could you modify your solution slightly with this hint? ✨ 
Let me know, if you would like to talk it more thoroughly through on Slack 😊